### PR TITLE
Modify firmware update documentation (3312857633)

### PR DESCRIPTION
**Pull request by pantsmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,48 @@ In general, you can send unlock commands as often and as quickly as you wish, be
 
 Failing to wait on setup can cause the lock to enter a state where it can't be setup until we reset it on the server. We have tried a number of different strategies to ease or prevent this issue, but as yet have not been 100% successful in all cases. Thus, it is in your best interest to specifically wait 60 seconds after the first unlock attempt of a new lock before trying again. This is also true after an old lock is reset (limited keys cleared from the database).
 
+### Updating Lock Firmware
+As with any other software the lock firmware needs updating from time to time. This can include for fixing bugs on the locks itself or for adding new functionality. Unfortunately, updating the lock firmware must be done one-by-one with each lock in-hand. There a number of reasons for this. The major reason is that there needs to be a stable Internet connection for downloading the new firmware directly to the lock.
+
```
Comment (sapsley) [inline README.md:136]: Look at removing the word "Unfortunately" to avoid negatively coloring the documentation

File: README.md [hunk 2]
```diff
@@ -131,6 +131,48 @@ In general, you can send unlock commands as often and as quickly as you wish, be
...
+#### Steps for updating firmware using the *Nokē Updater* app
+1. Install the *Nokē Updater*
+    * iOS
+        * Download *Nokē Updater* app from the Apple AppStore
+        * https://apps.apple.com/us/app/Nokē-update/id1270460855
+    * Android
+        * Download *Nokē Updater* app as an .apk found at the following link
```
Comment (sapsley) [inline README.md]: The name of the app is "Noke Update"

**Review by sapsley — CHANGES_REQUESTED** [review]
Added just a couple comments to look at before merging

File: README.md (hunk 2 above)
Comment (pantsmann) [reply README.md]: Thanks

**Review by sapsley — APPROVED** [review]
Looks good!

**Pull request merged**